app/results.py

```diff
@@ -111,12 +108,14 @@ def _consolidate_result_blob(self, blob_name):
         """
         try:
             # read the contents of the blob
-            blobContents = io.BytesIO()
-            self.storage_service.get_blob_to_stream(self.config.results_container, blob_name, blobContents)
-
-            # append the result blob contents to the consolidated file
-            self.logger.info("Appended results blob: " + blob_name)
-            self.append_storage_service.append_blob_from_stream(self.config.results_container, self.config.results_consolidated_file, blobContents)
+            with io.BytesIO() as blobContents:
+                self.storage_service.get_blob_to_stream(self.config.results_container, blob_name, blobContents)
+                blobContents.write(b'\n')
```

Comment:
I though results are encrypted.. In that case \n is useless.

Reply:
The individual results/rows/records are encrypted. Not the consolidated result. Hence, we can add \n to each row we are appending.